src/test/java/com/saiko/bidmarket/user/service/DefaultUserServiceTest.java

```diff
@@ -456,38 +467,44 @@ void ItResponseUserBiddingSelectResponse() {
 
         UserBiddingSelectRequest request = new UserBiddingSelectRequest(0, 1, END_DATE_ASC);
 
-        Product product = Product.builder()
-                                 .title("감자 팜")
-                                 .description("zz")
-                                 .location("강원도")
-                                 .category(ETC)
-                                 .minimumPrice(15000)
-                                 .images(List.of("testUrl"))
-                                 .writer(writer)
-                                 .build();
+        Product product = Product
+            .builder()
+            .title("감자 팜")
+            .description("zz")
+            .location("강원도")
+            .category(ETC)
+            .minimumPrice(15000)
+            .images(List.of("testUrl"))
+            .writer(writer)
+            .build();
         long productId = 1L;
         ReflectionTestUtils.setField(product, "id", productId);
 
         User bidder = new User("제로", "image", "google", "1234", new Group());
         long userId = 1L;
         ReflectionTestUtils.setField(bidder, "id", userId);
 
-        Bidding bidding = Bidding.builder()
-                                 .product(product)
-                                 .bidder(bidder)
-                                 .biddingPrice(BiddingPrice.valueOf(20000))
-                                 .build();
+        Bidding bidding = Bidding
+            .builder()
+            .product(product)
+            .bidder(bidder)
+            .biddingPrice(20000)
+            .build();
 
         given(biddingRepository.findAllUserBidding(anyLong(), any(UserBiddingSelectRequest.class)))
             .willReturn(List.of(bidding));
 
         //when
-        List<UserBiddingSelectResponse> result = defaultUserService.findAllUserBiddings(1,
-                                                                                        request);
+        List<UserBiddingSelectResponse> result = defaultUserService.findAllUserBiddings(
+            1,
+            request
+        );
```

Comment:
```suggestion
        List<UserBiddingSelectResponse> result = 
            defaultUserService.findAllUserBiddings(1, request);
```
이부분도 이런식으로 컨벤션을 맞추면 더 가독성이 좋을것 같은데 어떠신가요

Reply:
함수 호출부 줄 바꿈 기준이 너무 다양해지지는 않을까요? 
만약에 해당 컨벤션을 적용하게 되면 = 뒤에 줄바꿈해도 길어져서 줄 바꿈이 일어나는 상황에서는 = 뒤에 줄 바꿈 안하고 인자를 내리게 되는 건가요?